doc/md/integrations/kubernetes/operators/declarative.mdx

```diff
@@ -1,42 +1,10 @@
 ---
-title: The Atlas Kubernetes Operator
-id: kubernetes-operator
-slug: /integrations/kubernetes/operator
+title: Kubernetes Operator - Declarative Workflow
+id: declarative
```

Comment:
Are you sure this does not conflict with any other ID? Maybe use `kubernetes-declarative`?